File: TicTacToe/TicTacToe.Angular/src/app/data/dto/game-result-dto.ts
export class GameResultDto {
    gameId: number = 0;
    isTie: boolean = false;
    winnerId: number = 0;
    gameConnectionIds: Array<string> = [];

    constructor(gameId?: number, isTie?: boolean, winnerId?: number, gameConnectionIds?: Array<string>) {
        this.gameId = gameId ?? this.gameId;
        this.isTie = isTie ?? this.isTie;
        this.winnerId = winnerId ?? this.winnerId;
        this.gameConnectionIds = gameConnectionIds ?? this.gameConnectionIds;
    }
}

File: TicTacToe/TicTacToe.Shared/Dto/GameResultDto.cs
using System.Collections.Generic;

namespace TicTacToe.Api.Shared.Dto
{
    public class GameResultDto
    {
        public int GameId { get; set; }

        public bool IsTie { get; set; }

        public int WinnerId { get; set; }

        public List<string> GameConnectionIds { get; set; }
    }
}
